app/packs/entrypoints/application.scss

```diff
@@ -0,0 +1,5 @@
+$color-background: #efefef;
```

Comment:
I don't think we need these styles anymore after we've tested that this works. :-)

And do we still need `app/assets/stylesheets/application.css` and `app/assets/stylesheets/welcome.scss`?

Reply:
Done